Cython/Compiler/Builtin.py

```diff
@@ -450,3 +450,57 @@ def init_builtins():
 
 
 init_builtins()
+
+##############################
+# Support for a few standard library modules that Cython understands (currently typing and dataclasses
```

Comment:
Maybe want to close this parenthesis 😅

Reply:
I've fixed this locally - thanks. Don't think it's worth pushing it and forcing the tests to be rerun until I have some more substantial changes (especially given that master is currently broken).

But it'll be sorted in the next proper update!